spec/requests/api/v1/users_spec.rb

```diff
@@ -32,4 +32,28 @@ def create_call(user_params)
       it { expect(response).to have_http_status(:created) }
     end
   end
+
+  describe 'PATCH /index' do
+    def update_user_call(user_params)
+      patch api_v1_user_path(user), params: { user: user_params }
+    end
+
+    context 'when is succesful' do
+      let(:user_params) { { email: user.email, password: '123456' } }
+
+      before { update_user_call(user_params) }
+
+      it { expect(response).to have_http_status(:success) }
+    end
+
+    context 'when params are invalid' do
```

Comment:
I think that when params are invalid the http corresponding code should be :bad_params what do you think?

Reply:
I didn't find this code here: [HTTP status code symbols for Rails](https://gist.github.com/mlanett/a31c340b132ddefa9cca )
Also, unprocessable_entity indicates that the server understands the request but it is unable to process it.